conda/lock.py

```diff
@@ -20,13 +20,31 @@
 import logging
 import os
 import time
+from glob import glob
+from os.path import abspath, isdir, join
 
+from conda.install import rm_rf
+from .compat import range
 from .exceptions import LockError
 
 LOCKFN = '.conda_lock'
 
+# Keep the string "LOCKERROR" in this string so that external
+# programs can look for it.
+LOCKSTR = """\
```

Comment:
I pulled `lockstr` out of the actual class so that we don't generate a new string object every time we make a `Locked` object.